llvm/include/llvm/SYCLLowerIR/ESIMD/LowerESIMD.h

```diff
@@ -77,8 +77,8 @@ class SYCLLowerESIMDKernelPropsPass
 };
 
 // Fixes ESIMD Kernel attributes for wrapper functions for ESIMD kernels
-class SYCLLowerESIMDKernelAttrPass
-    : public PassInfoMixin<SYCLLowerESIMDKernelPropsPass> {
+class SYCLFixupESIMDKernelWrapperAttrPass
```

Comment:
Not a good name either, my bad. As I noted, sycl_explicit_simd is a metadata. So please rename once again - `SYCLFixupESIMDKernelWrapperMDPass`. We need another iteration anyways.